cmd/incus/network.go

```diff
@@ -915,6 +1007,13 @@ func (c *cmdNetworkList) Command() *cobra.Command {
 	cmd.RunE = c.Run
 	cmd.Flags().StringVarP(&c.flagFormat, "format", "f", "table", i18n.G("Format (csv|json|table|yaml|compact)")+"``")
 
+	cmd.ValidArgsFunction = func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		if len(args) != 0 {
+			return nil, cobra.ShellCompDirectiveNoFileComp
+		}
+
+		return c.global.cmpRemotes()
+	}
```

Comment:
Missing line break here

Reply:
Fixed. And also fixed my dependencies so I the checks locally.